src/generated/apiFetcher.ts

```diff
@@ -145,6 +145,12 @@ async function loadJob(signal: AbortSignal | undefined, delayedJobId: string): P
 
     const url = resolveV3Url(`/jobs/v3/delayedJobs/${delayedJobId}`);
     response = await fetch(url, { signal, headers });
+
+    if (response.status === 502 && retries > 0) {
```

Comment:
Please add a comment explaining why we're doing this; if I were a developer coming across this code in 6 months, I'd be pretty confused about why this is here.

Reply:
Great, a comment is added.